projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * WP Admin page with information and configuration shared among all Jetpack stand-alone plugins
+ *
+ * @package automattic/my-jetpack
+ */
+
+namespace Automattic\Jetpack\My_Jetpack;
+
+use Automattic\Jetpack\Admin_UI\Admin_Menu;
+
+/**
+ * The main Initializer class that registers the admin menu and eneuque the assets.
+ */
+class Initializer {
+
+	/**
+	 * Whether My Jetpack was already initialized.
+	 *
+	 * @var boolean
+	 */
+	private static $initialized = false;
```

Comment:
Maybe we could use an action to only init once? It's something I try to keep in mind that was mentioned to me [here](https://github.com/Automattic/jetpack/pull/19820#discussion_r631345925).

Reply:
I've seen both patterns and I usually prefer the static property, just because it's much more performant. I can see some advantages that @zinigor points out, but at the end of the day it's all theoretical, not sure we ever used these actions for anything else... Let me think this over... If anyone has strong opinions on that let me know. Not a big deal to me

Reply:
> not sure we ever used these actions for anything else

It just came to mind since we will be requiring all Jetpack plugins to add this. Exactly what/why someone might hook into that for? 🤷  but it sounds good in theory 😃

Reply:
ok, changed to the hooks approach, I could argue on the cons of this approach, or why it's not effective (we should add two hooks, one before and one after initialization, and offer a way for people to short circuit the initialization if we wanted this to be really useful).

But I like how explicit it is about what's going on, and easier to test